client/app/models/ProfileModel.js

```diff
@@ -4,14 +4,14 @@ export const Profile = Immutable.Record({
   familyName: 'family name',
   givenName: 'given name',
   description: 'product author',
-  avatarURL: null,
+  avatarImage: null,
```

Comment:
This could be set to correct value ```new Image()``` - makes it easier to understand what this record can hold.